Editor/Editor.cpp

```diff
@@ -6,6 +6,11 @@
 #include "Translator.h"
 #include "DummyVisualizer.h"
 
+#include <string>
```

Comment:
these includes should go to stdafx.h if they are not already there

Reply:
I've checked the original source and #include "Translator.h"
#include "DummyVisualizer.h" are already in Editor.cpp should I still change them or only the string include to stdafx.h?